modular_skyrat/modules/deployables/code/deployable.dm

```diff
@@ -0,0 +1,736 @@
+	update_appearance(UPDATE_ICON)
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/Destroy()
+	. = ..()
+	UnregisterSignal(src, COMSIG_ATOM_DIR_CHANGE)
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/update_overlays()
+	. = ..()
+	. += emissive_appearance(icon, "up-em", alpha = src.alpha)
+
+/obj/structure/bed/borg_action_pacifier/emag_act(mob/clicker)
+	if(obj_flags & EMAGGED)
+		return
+	to_chat(clicker, span_notice("You activate a sequence remnant from early development, initiating a combat program once the device deploys."))
+	clicker.log_message("emagged [src], activating its combat mode.", LOG_GAME)
+	undeploy()
+	do_sparks(2, TRUE, src)
+	obj_flags |= EMAGGED
+
+/obj/item/grenade/borg_action_pacifier_grenade/emag_act(mob/clicker)
+	if(obj_flags & EMAGGED)
+		return
+	to_chat(clicker, span_notice("You activate a sequence remnant from early development, initiating a combat program once the device deploys."))
+	clicker.log_message("emagged [src], activating its combat mode.", LOG_GAME)
+	do_sparks(2, TRUE, src)
+	obj_flags |= EMAGGED
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/user_buckle_mob(mob/living/target, mob/user, check_loc)
+	if(!do_after(target, 1.5 SECONDS, src))
+		return
+	..()
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/post_buckle_mob(mob/living/target)
+	if(!iscyborg(target))
+		target.pixel_y = (target.base_pixel_y + 18)
+		return
+	var/mob/living/silicon/robot/buckled_cyborg = target
+	if(R_TRAIT_TALL in buckled_cyborg.model.model_features)
+		buckled_cyborg.pixel_y = (buckled_cyborg.base_pixel_y + 18)
+	if(R_TRAIT_SMALL in buckled_cyborg.model.model_features)
+		buckled_cyborg.pixel_y = (buckled_cyborg.base_pixel_y + 12)
+	else if(!((R_TRAIT_SMALL || R_TRAIT_TALL) in buckled_cyborg.model.model_features))
+		buckled_cyborg.pixel_y = (buckled_cyborg.base_pixel_y + 16)
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/post_unbuckle_mob(mob/living/target)
+	if(!iscyborg(target))
+		target.pixel_y = initial(target.pixel_y)
+		return
+
+	target.pixel_y = target.base_pixel_y + target.body_position_pixel_y_offset
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/CanAttack(the_target)
+	if(ismob(the_target)) //Can't kick up! :-)
+		var/mob/target = the_target
+		if(target in buckled_mobs)
+			return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/death(gibbed)
+	if(power_storage && power_storage >= 5000)
+		explosion(src, -1, (power_storage / 12500), (power_storage / 12500), (power_storage / 10000), -1)
+	QDEL_NULL(particles)
+	return ..()
+
+/mob/living/simple_animal/hostile/borg_action_pacifier/proc/check_smoke()
+	if(power_storage)
+		if(power_storage >= 5000)
+			particles = new /particles/smoke/robot/BAPer/mild()
+		if(power_storage >= 20000)
+			particles = new /particles/smoke/robot/BAPer/bad()
+		if(power_storage >= 35000)
+			particles = new /particles/smoke/robot/BAPer/full()
+		else if (power_storage < 5000)
+			return
```

Comment:
Maybe this can be a switch?
Also you don't need to add ``if return``, just do nothing and it'll return.